packages/KPagination/KPagination.vue

```diff
@@ -157,6 +168,18 @@ export default {
     testMode: {
       type: Boolean,
       default: false
+    },
+    paginationType: {
+      type: String,
+      default: 'default'
+    },
```

Comment:
We should add a `PropType` interface here for the `beta` branch port

Reply:
I've added this to my beta branch port 👍